djangoproject/static/js/mod/search-key.js

```diff
@@ -17,8 +17,14 @@ define([
 
                 $(window).keydown(function(e) {
                     if ((e.metaKey || e.ctrlKey) && e.key === 'k' && $('input:focus, textarea:focus').length === 0) {
+                        const warning_banner_height = $('#dev-warning').outerHeight() || 0;
```

Comment:
There's also a banner when the version is [outdated](https://docs.djangoproject.com/en/1.8/), but it uses a different id (`#outdated-warning`). I would suggest using the class instead:
```suggestion
                        const warning_banner_height = $('.doc-floating-warning').outerHeight() || 0;
```

Reply:
Thanks @bmispelon for the suggestion. Apologies for responding so late, I must have missed this comment.

Reply:
Since search-key.js was [refactored](https://github.com/django/djangoproject.com/pull/1933) into `djangoproject.js`, I have make the suggested changes in the designated place. Please re-review. 🙌